CodeGeneration/Sources/generate-swiftsyntaxbuilder/SyntaxExpressibleByStringInterpolationConformancesFile.swift

```diff
@@ -0,0 +1,90 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import SwiftSyntax
+import SyntaxSupport
+import SwiftSyntaxBuilder
+import Utils
+
+let syntaxExpressibleByStringInterpolationConformancesFile = SourceFile {
+  ImportDecl(
+    """
+    \(raw: generateCopyrightHeader(for: "generate-swiftsyntaxbuilder"))
+    import SwiftSyntax
+    """)
+  ImportDecl("import SwiftParser")
+  ImportDecl("import SwiftParserDiagnostics")
+  
+  ExtensionDecl("extension SyntaxParseable") {
+    InitializerDecl(
+      """
+      public init(stringInterpolationOrThrow stringInterpolation: SyntaxStringInterpolation) throws {
+        self = try performParse(source: stringInterpolation.sourceText, parse: { parser in
+          return Self.parse(from: &parser)
+        })
+      }
+      """)
+  }
+  
+  for node in SYNTAX_NODES where node.parserFunction != nil && node.isBase {
+    if node.parserFunction != nil && node.isBase {
```

Comment:
I think this is always true because you’re checking it in the loop condition. Which means you can also remove the `else` branch.

Reply:
Ah, makes sense 👍